### PR TITLE
CORE-5301 removing corda osgi lib from plugins

### PR DESCRIPTION
**Pull request by pmck91**
Removing unnecessary dependency on the `net.corda.kotlin:kotlin-stdlib-jdk8-osgi` lib

### CODE REVIEW

**Review by chrisr3 — COMMENTED** [review]
Aren't these plugins written in Kotlin? So wouldn't you want to include the JDK8 extension to the Kotlin standard library? Without it, you're at best constrained only to use Kotlin JDK6-compatible APIs.

For reference, `kotlin-stdlib-jdk8-osgi` is just `kotlin-stdlib-jdk8` with added OSGi metadata.

**Comment by benmcmahon100** [comment]
> Aren't these plugins written in Kotlin? So wouldn't you want to include the JDK8 extension to the Kotlin standard library? Without it, you're at best constrained only to use Kotlin JDK6-compatible APIs.
> 
> For reference, `kotlin-stdlib-jdk8-osgi` is just `kotlin-stdlib-jdk8` with added OSGi metadata.

Yeah these were the wrong ones though and there's a WIP PR to add those as a common dependency in the cli host

**Comment by pmck91** [comment]
> Aren't these plugins written in Kotlin? So wouldn't you want to include the JDK8 extension to the Kotlin standard library? Without it, you're at best constrained only to use Kotlin JDK6-compatible APIs.
> 
> For reference, `kotlin-stdlib-jdk8-osgi` is just `kotlin-stdlib-jdk8` with added OSGi metadata.

do we need the OSGI version? or can I swap to compile time dep in the plugins and have the host supply the runtime version?

File: tools/plugins/secret-config/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ group 'net.corda.cli.deployment'
 
 dependencies {
     implementation platform("net.corda:corda-api:$cordaApiVersion")
```
Comment (chrisr3) [inline tools/plugins/secret-config/build.gradle:11]: Do you still need the `platform()`, now that you've removed `kotlin-stdlib-jdk8-osgi`?

Comment (pmck91) [reply tools/plugins/secret-config/build.gradle:11]: it fails to build without it

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**